.travis.yml

```diff
@@ -60,3 +61,4 @@ notifications:
 
 # Route your build to the container-based infrastructure for a faster build
 sudo: false
+
```

Comment:
Redundant. We just need single blank line at the end.

Reply:
Yeah! removed that